app/models/resource.py

```diff
@@ -0,0 +1,30 @@
+from .. import db
+
+
+class Resource(db.Model):
+    __tablename__ = 'resources'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64))
+    address = db.relationship('Address', backref='resource',
```

Comment:
Do you want the address every time you get a resource? If so, I would make this a `joined` load.